docs/getting-started/installation/mac.md

```diff
@@ -6,102 +6,58 @@ sidebar_position: 1
 
-```shell
-brew install nvm
-nvm install $(cat .nvmrc)
-```
-
-You can also manage your NPM with [asdf](https://nodecli.com/nodejs-asdf).
+Use `rtx install` - this will pull the exact versions we recommend for the tools listed in the block above, which helps to prevent "version drift" - where someone writes code targeting Version A of some tool, but it breaks when run against Version B of that same tool. **This step will probably take several minutes**: it's building most of these dependencies from source code. Errors at this point are unlikely, but if they pop up, should be expected to be quite varied: searching for error messages will often lead to helpful solutions, as few to none of these errors are likely to have never been seen (and triaged) before.
 
-## Installing Forem
+## Last Bits Of Setup
 
-1. Fork Forem's repository, e.g. <https://github.com/forem/forem/fork>
-2. Clone your forked repository in one of two ways:
+1. Start the caching server with `redis-server --daemonize yes` (do this every time you work on Forem!)
 
-   - e.g. with HTTPS: `git clone https://github.com/<your-username>/forem.git`
-   - e.g. with SSH: `git clone git@github.com:<your-username>/forem.git`
+2. Start and configure the database (which requires some tinkering to avoid future errors; copy pasting is fine here!)
 
-3. Install bundler with `gem install bundler`
+  - `pg_ctl start` - do this every time you work on Forem!
+  - `createuser -U postgres -s $(whoami)`, which creates a PostgreSQL user by your system username. This only needs done once.
+  - Now we need to change the default encoding of the database. These steps are copied nearly verbatim from [the Arch Linux wiki](https://wiki.archlinux.org/title/PostgreSQL#Change_default_encoding_of_new_databases_to_UTF-8). Run them one-by-one.
 
-   - **Note:** If you are using a Mac with an M1 CPU that uses Ruby compiled for ARM, you may need to force the platform that Bundler uses to `ruby`. This will allow you to build the C extensions used by Forem's dependencies.
-To force the platform to use `ruby`, you can either:
-     - set `BUNDLE_FORCE_RUBY_PLATFORM=true` in the shell environment
-     - run `bundle config set force_ruby_platform true` to set `ruby` in Bundler globally
+    * `psql -d postgres -c "UPDATE pg_database SET datistemplate = FALSE WHERE datname = 'template1';"`
+    * `psql -d postgres -c "DROP  DATABASE template1;"`
+    * `psql -d postgres -c "CREATE DATABASE template1 WITH TEMPLATE = template0 ENCODING = 'UNICODE';"`
+    * `psql -d postgres -c "UPDATE pg_database SET datistemplate = TRUE WHERE datname = 'template1';"`
 
-4. Set up your environment variables/secrets
+3. Set up your environment variables/secrets
```

Comment:
Since this was already part of the docs, I think I'll vote to fix that in a future following PR if we so desire (also in part because I'm less versed in the docs we're linking out to there :sweat_smile:)